wandb/filesync/dir_watcher.py

```diff
@@ -291,7 +291,10 @@ def _get_file_event_handler(
                 )
             else:
                 make_handler = PolicyEnd
-                for policy, handler_type in [("now", PolicyNow), ("live", PolicyLive)]:
+                for policy, handler_type in [("live", PolicyLive), ("now", PolicyNow)]:
+                    # TODO(spencerpearson): ^ that list should be in the _other_ order:
```

Comment:
if you think this is the preferred behavior I think it is fine to change it. I don't think we have any guarantees on multiple policies. I would only check the documentation if there is nothing it is fine to change

Reply:
Hmm, I'll leave the behavior alone for now, to keep this PR purely-test-and-refactor. I can issue a followup PR to clean up these behavior-changing TODOs. Sound okay?

Reply:
Love the incremental changes! My favorite PRs small and fixing one thing at a time :)